### DISCUSSION TITLE
Node v20 not available despite Docs saying so

### DISCUSSION THREAD

**Discussion by felixmpaulus** [vercel/community#4876] [Bug Report]
### Page to Investigate

https://vercel.com/docs/functions/serverless-functions/runtimes/node-js#node.js-version

### Steps to Reproduce

I require node version 20 to build my project. 
The docs say, that version 20 is available in beta since yesterday, November 15th 2023.
But in my project settings only version 16 and 18 are selectable.

<img width="993" alt="Screenshot 2023-11-16 at 22 47 41" src="https://github.com/vercel/community/assets/35754650/f37f1304-7f05-46d4-91aa-f77f47059531">

**Comment by TooTallNate** [comment]
Ha! You're too quick 😉 Try refreshing the page, we have now rolled out Node.js v20 for all users.

**Reply by felixmpaulus** [reply]
Wow what a timing. 
As a matter of fact I switched a project from Heroku to Vercel just today and thought to myself "what a timing that they enabled node 20 just yesterday".
Thank you for the quick response on this!
